webapp/sources/rudder/rudder-web/src/main/webapp/techeditor/js/ncf.js

```diff
@@ -1634,10 +1633,6 @@ $scope.onImportFileChange = function (fileEl) {
                 delete ct[i].agent_support;
                 delete st[i].promiser;
                 delete ct[i].promiser;
-                if(st[i].OS_class.minorVersion === undefined) delete st[i].OS_class.minorVersion;
-                if(st[i].OS_class.majorVersion === undefined) delete st[i].OS_class.majorVersion;
-                if(ct[i].OS_class.minorVersion === undefined) delete ct[i].OS_class.minorVersion;
-                if(ct[i].OS_class.majorVersion === undefined) delete ct[i].OS_class.majorVersion;
```

Comment:
This lines were added in #17776 to correct the bug with "change detected" pop-up. I don't think we can remove them.
(see commit:9c416c8c9515e956fd45be8d5c919055e8a51a6f) cc @VinceMacBuche